src/aics-image-viewer/components/App/types.ts

```diff
@@ -1,8 +1,9 @@
-import type { RawArrayData, RawArrayInfo, Volume } from "@aics/volume-viewer";
+import type { RawArrayData, RawArrayInfo, View3d, Volume } from "@aics/volume-viewer";
 
 import type { MetadataRecord } from "../../shared/types";
 import type { ViewerChannelSettings } from "../../shared/utils/viewerChannelSettings";
 import type { ViewerState } from "../ViewerStateProvider/types";
+import { MutableRefObject } from "react";
```

Comment:
nit: should be sorted? I think you only implemented auto sorting imports on the Timelapse app